sql/catalyst/src/main/java/org/apache/spark/sql/connector/write/BatchWrite.java

```diff
@@ -45,8 +45,10 @@
    *
    * If this method fails (by throwing an exception), the action will fail and no Spark job will be
    * submitted.
+   *
+   * @param info Information about the RDD that will be written to this data writer
```

Comment:
nit: `Physical information about the input data that will be written to this data writer.`